src/components/scenes/SendConfirmationScene.js

```diff
@@ -38,6 +39,7 @@ const DIVIDE_PRECISION = 18
 export type SendConfirmationStateProps = {
   fiatCurrencyCode: string,
   currencyInfo: EdgeCurrencyInfo | null,
+  currencySettings: CurrencySetting,
```

Comment:
Why is this here?

Reply:
Originally it needed it when it was the main screen.